todo_app/app.py

```diff
@@ -105,6 +123,8 @@ def new_card():
     def view_card(cardId):
         card = items.get_card(cardId)
 
+        app.logger.info("Current card: %s", card[0]['name'])
```

Comment:
nitpick: You could use nice modern f-string formatting here and elsewhere `f"Current card: {card[0]['name']}"`